gosuper/gosuper/api.go

```diff
@@ -236,6 +237,15 @@ func LogToDisplayControl(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	if loaded, err := systemd.Dbus.GetUnitProperty(serviceName, "LoadState"); err != nil {
+		sendError(fmt.Errorf("Unable to get log to display load status: %v", err))
+		return
+	} else if loaded.Value.String() == `"not-found"` {
```

Comment:
Yeah, my worry is running this on a system that isn't using English strings, or just if systemd decides to reword the outputs - if there's no avoiding it then I can live with it, but would rather avoid it if possible